packages/core/src/index.ts

```diff
@@ -164,7 +166,7 @@ export function buildSendMail<T>(options: BuildSendMailOptions<T>) {
       log("💌 opening sendMail preview", debugMail);
       // create an intercept on the preview server
       // then open it in the browser
-      const PREVIEW_SERVER_URL = "http://localhost:3883/intercepts";
+      const PREVIEW_SERVER_URL = `${previewServerUrl?.trim().replace(/\/+$/, "") || "http://localhost:3883"}/intercepts`;
```

Comment:
This regex replaces trailing slashes so this doesn't break if they are included.

Demo: regexr.com/73m23